common/types.d.ts

```diff
@@ -239,6 +239,10 @@ declare global {
       color?: number
       label?: string
       accessDisabled?: boolean
+      onlineDeviceNotification?: boolean
+      onlineSharedDeviceNotification?: boolean
```

Comment:
I don't understand what these two are for. At the device level we should only need notifications for "Email" or "System"
```
      onlineDeviceNotification?: boolean
      onlineSharedDeviceNotification?: boolean
```